app/views/forms/show.xml.erb

```diff
@@ -13,6 +13,13 @@
             <data id="<%= @form.id %>" version="<%= @form.current_version.sequence %>">
           <% end %>
 
+          <%# group name and notes %>
+          <% groups = QingGroup.child_groups(@form.root_group.children) %>
```

Comment:
`children` is delegated to root_group on form, so you can just do @form.children. LoD says you want to avoid chaining method calls wherever possible. So always look for delegators and create them where appropriate.